star-ros2/src/star_spi_bridge/include/star_spi_bridge/spi_message_converter.hpp

```
@@ -1,7 +1,7 @@
 // Copyright 2026 Locked Inc.
 
-#ifndef STAR_SPI_BRIDGE_INCLUDE_STAR_SPI_BRIDGE_SPI_MESSAGE_CONVERTER_HPP_
-#define STAR_SPI_BRIDGE_INCLUDE_STAR_SPI_BRIDGE_SPI_MESSAGE_CONVERTER_HPP_
+#ifndef STAR_SPI_BRIDGE__SPI_MESSAGE_CONVERTER_HPP_
+#define STAR_SPI_BRIDGE__SPI_MESSAGE_CONVERTER_HPP_
```

Clang (14.0.6)

[warning] declaration uses identifier 'STAR_SPI_BRIDGE__SPI_MESSAGE_CONVERTER_HPP_', which is a reserved identifier

(bugprone-reserved-identifier)